Actualizacion del footer

AdolfoHMtz opened the pull request:
footer actualizado

AdolfoHMtz committed:
Update index.html

footer actualizado

AdolfoHMtz committed:
Update index.html

Rolexus934 commented:
Intenta cambiar la distribución del grid por un 2-2-2-2 para que se vea centrado, también elimina el breakpoint lg, ya que en pantallas 720p cambiara la resolución al modo tableta

Rolexus934 committed:
Centrado contenido del Footer

Se centro el footer para que no quedará espacio sobrante al final, se ve más bonito ;)

Rolexus934 committed:
Centrando contenido footer

nashe

Rolexus934 committed:
Centrar contenido footer

soy tonto

AdolfoHMtz committed:
Update index.html

hola links actualizados
